### DISCUSSION TITLE
Https

### DISCUSSION THREAD

**Discussion by axaaxa6520** [rejetto/hfs#442] [General]
Is there the possibility of also inserting the bundle ca to close the certification chain and have maximum compatibility?

**Comment by rejetto** [comment]
i've never done anything like that.
Is this what you are looking for?
> Optionally override the trusted CA certificates. Default is to trust the well-known CAs curated by Mozilla. Mozilla's CAs are completely replaced when CAs are explicitly specified using this option. The peer's certificate must be chainable to a CA trusted by the server for the connection to be authenticated. When using certificates that are not chainable to a well-known CA, the certificate's CA must be explicitly specified as a trusted or the connection will fail to authenticate. If the peer uses a certificate that doesn't match or chain to one of the default CAs, use the ca option to provide a CA certificate that the peer's certificate can match or chain to. For self-signed certificates, the certificate is its own CA, and must be provided. For PEM encoded certificates, supported types are "TRUSTED CERTIFICATE", "X509 CERTIFICATE", and "CERTIFICATE".
